### ISSUE TITLE
Ensure ldconfig can be found in CUDA easyblock

### ISSUE THREAD

**Issue by willfurnass** [easybuilders/easybuild-easyblocks#1680]
At present the `cuda.py` easyblock (v3.8.1) depends on being able to find and run `ldconfig`:

```python
    def post_install_step(self):
       # ...
       # Run ldconfig to create missing symlinks in the stubs directory (libcuda.so.1, etc)
       run_cmd("ldconfig -N " + os.path.join(self.installdir, 'lib64', 'stubs'))
```

However, on Centos 7x `ldconfig` is in `/sbin/` and `/usr/sbin/` but neither dir may be on the `PATH` e.g. in certain batch job environments.  

Should this case be handled in the easyblock?

**Comment by akesandgren** [comment]
The first question here is if we should just add /sbin:/usr/sbin to PATH before executing the ldconfig, or check if it can be found and bail out if not?
@boegel any comments on that?
I think adding /sbin:/usr/sbin and then checking it might be the best way here...

**Comment by akesandgren** [comment]
Proposed fix in https://github.com/easybuilders/easybuild-easyblocks/pull/1684

**Comment by boegel** [comment]
closing since #1684 just got merged, fix will be included in upcoming EasyBuild v3.9.1
